### PR TITLE
Rework the documentation around our cloud services

### PR DESCRIPTION
**Pull request by choldgraf**
After a recent conversation with @colliand and the CS&S folks, this is a re-working of the "pilot documentation" to prepare for an "alpha" launch of the hub service. It does the following major things:

- Uses the PyData Sphinx Theme so that we have more consistent style / structure with our `2i2c.org` website
- Splits this documentation into two major sections: one for hub Administrators, and one about the service itself.
- Adds more meta-information about the cloud services in general, a overview matrix of features, and pages for each of the two major hub distributions that we offer.
- Adds a bit more build infrastructure to make it easier to build the docs locally.

Here's a quick GIF of the header of this site compared with 2i2c.org, along with the changes also proposed in https://github.com/2i2c-org/2i2c-org.github.io/pull/69

AE3D2mS5p2

In addition to this, I'd also like to make the following URL changes:

- [ ] Make `docs.2i2c.org` point to this site
  - [ ] The Hub Administrator guide would then be available at `docs.2i2c.org/admin`
  - [ ] The "About our Service" guide would be available at `docs.2i2c.org/about`
- [ ] Make `devops.2i2c.org` point to our current `pilot-hubs` documentation

part of https://github.com/2i2c-org/meta/issues/78

### CODE REVIEW

File: about/overview.md [hunk 1]
```diff
@@ -1,20 +1,9 @@
 (about-the-project)=
-# About the Managed JupyterHub Service
+# Overview
 
 2i2c JupyterHubs offer data science environments to multiple users via a JupyterHub that runs in the cloud.
```
Comment (colliand) [inline about/overview.md:4]: I expect some of our customers will not identify as belonging to "data science". I suggest we say "2i2c JupyterHubs offer interactive computing environments designed to serve multiple users via a JupyterHub that runs in the cloud."

File: about/services/education.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+(hub-types:education)=
+# Collaborative learning with data science
+
+:::{admonition} TODO
+This section is incomplete, it needs more information!
+:::
+
+The 2i2c Educational Hubs provide standard data science environments and infrastructure that is meant for teaching data science. They are inspired by 2i2c's experience with the [DataHubs at UC Berkeley](https://docs.datahub.berkeley.edu/en/latest/).
```
Comment (colliand) [inline about/services/education.md:8]: "....provide standard interactive computing environments and infrastructure designed for teaching data science. These hubs are inspired by 2i2c's experience with the [DataHubs at UC Berkeley](https://docs.datahub.berkeley.edu/en/latest/) and the [Syzygy service](https://syzygy.ca/) for Canada."

File: about/services/overview.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+:::{note}
+Our services are in an "alpha" state - we are still learning a lot about the best way that these hubs can serve communities in research and education.
+The infrastructure and service may change over the coming months!
+See [our strategy page](../strategy.md) for an overview of what we're hoping to do and where we're headed next.
+:::
+
+## What technology makes up each hub?
+
+🚀 core infrastructure
+: Underneath each 2i2c JupyterHub is a [JupyterHub](https://jupyter.org/hub). These provide interactive computing sessions for each of your users, and connect to the other infrastructure in the cloud. We use [`auth0`](https://auth0.com/) for authenticating users, which can connect to a number of other authentication protocols (such as OAuth2).
```
Comment (colliand) [inline about/services/overview.md:15]: ".....use [`auth0`](https://auth0.com/) and [CILogon](https://www.cilogon.org/) for authenticating...."

**Review by colliand — APPROVED** [review]
I like the changes! I added a few comments while reviewing. I don't think we are in final form but see these as a significant improvement. I suggest we merge and continue our work together.

File: about/services/education.md (hunk 2 above)
Comment (choldgraf) [reply about/services/education.md]: done 👍

**Comment by choldgraf** [comment]
I made another update after a recent conversation with @colliand. It took inspiration from the Strip documentation, which has been pointed out as a good example to follow. It makes the following changes:

- Re-works this site to be more of a high level documentation that points to other places. I imagine we can host this at `docs.2i2c.org`.
- Updates the logo to be further in-line with 2i2c.
- Includes links to our `Hub Engineer's guide` (which I'm calling the `DevOps Guide` since I suspect that will be more descriptive than Hub Engineer) as well as our Team Compass (since we believe that our transparency and good org structure is another "feature" of 2i2c)

would love feedback - particularly from @colliand if he thinks this is a step in the right direction. If so, and if nobody from the tech team objects, I suggest we merge this and iterate from there.

**Comment by choldgraf** [comment]
OK I think this is more-or-less ready to go. There's still a lot more to improve, but this is I think a good step. I'll leave it open for a day or two to see if others have ideas (or objections :slightly_smiling_face:)

**Review by sgibson91 — APPROVED** [review]
I really like this! Feels clean and easy to navigate

**Comment by choldgraf** [comment]
thanks @sgibson91 and @colliand for taking a look@!

**Comment by damianavila** [comment]
Late to the party but I really like this one :wink:

**Pull request merged**